### ISSUE TITLE
Is this project implementation of MulPIR？

### ISSUE THREAD

**Issue by Jumdar** [OpenMined/PIR#47]
## Question
**Describe your question in ONE SENTENCE.**
Is this project implementation of MulPIR？
## Further Information
Describe your question in greater length here.

## Screenshots
If applicable, add screenshots to help explain your question.

## System Information
 - OS Version: [e.g. 22]
 - Language Version: [e.g. Python 3.7, Node 10.18.1]
 - Package Manager Version: [e.g. Conda 4.6.1, NPM 6.14.1]

**Comment by RasoulAM** [comment]
Yes, this project includes an implementation of MulPIR. If you set the protocols appropriately, the resulting protocol is MulPIR. Specifically, you must set the ```USE_CIPHERTEXT_MULTIPLICATION``` to be ```true```. You can see try this by changing this flag in the benchmark.cpp file.
